dialog/src/main/java/roboy/dialog/states/botboy/BotBoyQuestionAnsweringState.java

```diff
@@ -0,0 +1,273 @@
+package roboy.dialog.states.botboy;
```

Comment:
what is new about this state? compared to ordinary and devo?

what sense does it make to create copies of almost identical states? @Waguramu you were requesting this i think

Reply:
@SezerCan

Reply:
Since we did not use each transition etc. that ordinary state has, having our own state was efficient and more safe for our demo. Also later on these states can be redesigned for more chat specific conversations without affecting other groups. @missxa @Waguramu

Reply:
If this is identical to other states, it doesn’t make sense

Reply:
@SezerCan please remove this state